### ISSUE TITLE
System.MissingMethodException: 'Method not found: 'Boolean FlyleafLib.Master.get_PreventAutoDispose()'.'

### ISSUE THREAD

**Issue by LiorBanai** [SuRGeoNix/Flyleaf#82]
Hi,
I'm trying to use the library and get the above exception:
Method not found: 'Boolean FlyleafLib.Master.get_PreventAutoDispose()'.

using :
```xml
    <PackageReference Include="FlyleafLib" Version="3.3.1" />
    <PackageReference Include="FlyleafLib.Controls.WPF" Version="1.1.7" />
```
![image](https://user-images.githubusercontent.com/36262995/132135419-c81f12b4-9737-45d5-a87c-b9cdea755614.png)

**Comment by SuRGeoNix** [comment]
Yep, that's bad... I've removed that attribute on v3.3.1 and I didn't update the WPF package. I will have soon the new release which will fix it. Until then you might downgrade the FlyleafLib to v3.3.0?

**Comment by LiorBanai** [comment]
Sure, I'll downgrade. 
thanks for the quick response :)

**Comment by SuRGeoNix** [comment]
Fixed in v3.3.2 / v.1.1.8
